operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [ENHANCEMENT] Update the `rollout-operator` subchart to `0.9.1`. #6110
 * [ENHANCEMENT] Update the `rollout-operator` subchart to `0.9.0`. #6022
```

Comment:
should we collapse the two entries into one?

Reply:
```suggestion
* [ENHANCEMENT] Update the `rollout-operator` subchart to `0.9.1`. #6022 #6110
```